src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1185,7 +1185,7 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
         [ActiveIssue(20010, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [Theory]
-        [InlineData(99)]
+        [InlineData(99)] // Doesn't work in UAP
```

Comment:
Maybe we can separate this out and add a new bug or pin this behavior with a separate UAP test.